### PR TITLE
Update docs for primary branch rename to "main"

### PR DESCRIPTION
**Pull request by sftim**
Part of the fixes for #28462

### CODE REVIEW

**Comment by sftim** [comment]
/hold

Rename has not yet happened

File: content/en/docs/contribute/participate/pr-wranglers.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ These queries exclude localization PRs. All queries are against the main branch
...
 - [Has LGTM, needs docs approval](https://github.com/kubernetes/website/pulls?q=is%3Aopen+is%3Apr+-label%3Ado-not-merge%2Fwork-in-progress+-label%3Ado-not-merge%2Fhold+label%3Alanguage%2Fen+label%3Algtm+):
   Lists PRs that need an `/approve` comment to merge.
-- [Quick Wins](https://github.com/kubernetes/website/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+base%3Amaster+-label%3A%22do-not-merge%2Fwork-in-progress%22+-label%3A%22do-not-merge%2Fhold%22+label%3A%22cncf-cla%3A+yes%22+label%3A%22size%2FXS%22+label%3A%22language%2Fen%22): Lists PRs against the main branch with no clear blockers. (change "XS" in the size label as you work through the PRs [XS, S, M, L, XL, XXL]).
-- [Not against the main branch](https://github.com/kubernetes/website/pulls?q=is%3Aopen+is%3Apr+label%3Alanguage%2Fen+-base%3Amaster): If the PR is against a `dev-` branch, it's for an upcoming release. Assign the [docs release manager](https://github.com/kubernetes/sig-release/tree/master/release-team#kubernetes-release-team-roles) using: `/assign @<manager's_github-username>`. If the PR is against an old branch, help the author figure out whether it's targeted against the best branch.
```
Comment (jimangel) [inline content/en/docs/contribute/participate/pr-wranglers.md:48]: This is an incredible catch :)

**Comment by jimangel** [comment]
/lgtm
/approve

(once CI catches up)

**Pull request merged**